pxtsim/runtime.ts

```diff
@@ -3,6 +3,14 @@
 namespace pxsim {
     const MIN_MESSAGE_WAIT_MS = 200;
     let tracePauseMs = 0;
+
+    enum MessageListenerFlags {
```

Comment:
Just wanted to put this here for reference: https://github.com/lancaster-university/codal-core/blob/509086cc8590465041b15493ab52b56e7071c110/inc/core/CodalListener.h#L33-L41

Also, just wanted to know why we're not including all of the defined flags here?

Reply:
those other ones are internal